codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -213,4 +137,96 @@ private static void insert(Node root, int symbol, int code, byte length) {
             current.children[i] = terminal;
         }
     }
+
+    private static final class DecoderProcessor implements ByteProcessor {
+        private final ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Yep

> Am 08.06.2016 um 21:06 schrieb Scott Mitchell <notifications@github.com>:
> 
> In codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java:
> 
> > @@ -213,4 +137,96 @@ private static void insert(Node root, int symbol, int code, byte length) {
> >              current.children[i] = terminal;
> >          }
> >      }
> > +
> > +    private static final class DecoderProcessor implements ByteProcessor {
> > +        private final ByteArrayOutputStream baos = new ByteArrayOutputStream();
> was the plan to implement a AppendableByteSequence (just like AppendableCharSequence)?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>